### DISCUSSION TITLE
Trying to create status for how long i am with each track in projects with templater

### DISCUSSION THREAD

**Discussion by Rasmusbkr** [SilentVoid13/Templater#1025] [Help]
Hello i'm completely fresh in this coding thing and I took an example from a yt video and executed that to the best of my knowledge and have made progress but can't seem to make the last part stick.

![Step 1](https://user-images.githubusercontent.com/124707961/217309838-011873ae-987f-4d11-8f16-5c08d762c7af.png)
![Step 2](https://user-images.githubusercontent.com/124707961/217309844-e018a38b-ac6b-4f2d-a702-50326da9bcee.png)
![step 3 error](https://user-images.githubusercontent.com/124707961/217309845-920a057a-d2c0-4690-b703-d1cbd1b9a08f.png)

I get the same error every time but now i at least get to pick the status but it still fails :/ 
Any help is greatly appreciated :)

**Comment by welpdx** [comment]
The error seems to say that it is because you have a duplicate "status" entry in the yaml. Not sure if that is the real problem though. 

I think the real error is the "<" special character. What happens if you enclose the whole statement with quotes? ie `status: "<%tp.system.suggester(...) %>"` ?

Anyways, I could not replicate the error that you have. After the prompt, the text is then populated in the note and that's it, there is no error in the console about yaml parsing error.  Btw, what is this Project Inspector? Are there other plugins installed?

**Reply by Rasmusbkr** [reply]
Still getting the same error, but thank you for giving it a shot.
I have provided screenshots of all my plugins and the "Projects" plugin more in depth.
![projects](https://user-images.githubusercontent.com/124707961/217830963-02639203-5280-4f83-adf3-0542e76645c0.png)
![full plugins](https://user-images.githubusercontent.com/124707961/217830985-ab6e2d85-381a-4eba-b78d-a82798f2f6c3.png)

**Reply by Rasmusbkr** [reply]
I figured it out "Status: " can't be listed twice 👍
